### ISSUE TITLE
Setup for other ESP32 boards.

### ISSUE THREAD

**Issue by choffee** [mfikes/esprit#33]
I'm trying to get things up and running on another board. ( TTGO ESP32-WROVER ) but it's giving me errors on boot. The erase/bootstrap/flash seems to work but once on the serial console it gives the following error a few times then reboots:

invalid header: 0x69726124

My suspicion is that it's the board config clashing with the pins in some way. I just can't work out how to change the defaults. I've created a new board config that might work but can't work how to set the "env" to specify it's location. 

Should this just work on other ESP32 boards or have I got the wrong end of things? ( I may be a little out my depth here :)

**Comment by gdanov** [comment]
isn't that board with 4mb flash? the manual says you need 8mb.

**Comment by choffee** [comment]
I think you might be right. I may have misread the manual where it said  "The ESP32 WROVER module with 8MB of PSIRAM has been shown to work great. " I think mine has 8MB of PSIRAM which I think mine has but only 4Mb of Flash as you say.

**Comment by gdanov** [comment]
Yes. I was looking to buy device myself these days and it took me some time to realize  the minimum requirement is for the Flash. In the clojurians slack this was confirmed by MFikes in reply to someone else also on the hunt.
